### DISCUSSION TITLE
which is faster, searching via css or xPath?

### DISCUSSION THREAD

**Discussion by sweetppro** [tid-kijyun/Kanna#262] [Ideas]
Im wanting to speed up my html parsing.
Is there a preferred method fast fastar parsing?
also, are there any benefits of one over the other.
or perhaps is is best to use both

any insight would be great,
thanks

**Comment by tid-kijyun** [comment]
XPath is faster than CSS. Since CSS converts to XPath and executes as XPath.
If you want to do it repeatedly in one process, I recommend using XPath. In other cases, there is almost no difference in speed, so use whichever you like.

**Reply by sweetppro** [reply]
Thanks
And thanks for this great Framework.
